doc/source/changes/version_0_30.rst.inc

```diff
@@ -135,6 +135,70 @@ New features
     a_b  a0_b1  a1_b2
              1      5
 
+* implemented :py:obj:`LArray.keys()` :py:obj:`LArray.values()` and :py:obj:`LArray.items()`
+  methods to iterate (loop) on an array labels (keys), values or (key, value) pairs.
+
+    >>> arr = ndtest((2, 2))
+    >>> arr
+    a\b  b0  b1
+     a0   0   1
+     a1   2   3
+
+    By default they iterates on all axes, in the order they are in the array.
+
+    >>> for value in arr.values():
+    ...     print(value)
+    0
+    1
+    2
+    3
+    >>> for key in arr.keys():
+    ...     print(key)
```

Comment:
print(key, '->', label) ? 
Some users may want to see how to convert single IGroup to the corresponding label.

Reply:
I tried to come up with a temporary fix in 6007903, do you think that's enough for now?